wdio.conf.cjs

```diff
@@ -82,6 +82,7 @@ module.exports.config = {
     if (browserName === 'chrome') {
       capability['goog:chromeOptions'] = {
         excludeSwitches: ['enable-automation'],
+        args: ['disable-search-engine-choice-screen'],
```

Comment:
This is included in puppeteer 21.2.1 so could skip this and wait for that to come through. It doesn't cause issues, other than being annoying when testing locally.